modules/ROOT/pages/save-builtin-dashboard-as-custom.adoc

```diff
@@ -0,0 +1,25 @@
+= Save a Built-in Dashboard as a Custom Dashboard
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To simplify the process of creating a custom dashboard from scratch, you can save any of the built-in dashboards that are included in the Anypoint Monitoring tabs as a base template for a custom dashboard that you can then customize to suit your needs. This allows you to take advantage of the flexibility and features offered in custom dashboards, such as troubleshooting, setting up alerts, or comparing multiple resources within one dashboard. 
```

Comment:
delete "from scratch" it's colloquial and also implied in "custom." Also, do you really create a template and then a custom dashboard? The task instructions just have you creating one thing, a new custom dashboard.

Reply:
They are saving a built-in dashboard with all its charts and graphs as a custom dashboard, so it's a template because it already has stuff in it. They can't customize built-in dashboards. So being able to save a built-in dashboard as a custom dashboard lets them customize it however they want.